include/crow/app.h

```diff
@@ -68,7 +68,7 @@ namespace crow
         }
 
         /// Process the request and generate a response for it
-        void handle(const request& req, response& res)
```

Comment:
I removed const qualifiers up to a rules handle to enable mutable requests in local middleware